projects/demo/src/modules/components/dialog-experimental/index.ts

```diff
@@ -0,0 +1,74 @@
+import {Component, inject, type TemplateRef} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {DemoRoute} from '@demo/routes';
+import {TuiDemo} from '@demo/utils';
+import {TuiAutoFocus} from '@taiga-ui/cdk';
+import {
+    TuiAlertService,
+    TuiButton,
+    type TuiDialogContext,
+    TuiTextfield,
+} from '@taiga-ui/core';
+import {TUI_DIALOG_OPTIONS, TuiDialogService} from '@taiga-ui/experimental';
+import {TuiForm} from '@taiga-ui/layout';
+import {switchMap} from 'rxjs';
+
+@Component({
+    standalone: true,
+    imports: [TuiAutoFocus, TuiButton, TuiDemo, TuiForm, TuiTextfield],
+    templateUrl: './index.html',
+    changeDetection,
+})
+export default class Page extends Array {
+    private readonly alerts = inject(TuiAlertService);
+    private readonly dialogs = inject(TuiDialogService);
+    private readonly options = inject(TUI_DIALOG_OPTIONS);
+
+    protected readonly routes = DemoRoute;
+    protected readonly examples = [
+        'String',
+        'Directive',
+        'Component',
+        'Closing',
+        'Customization',
+    ];
```

Comment:
Why this example was not copied too ?

https://taiga-ui.dev/components/dialog#confirm